go/vt/vttablet/tabletmanager/rpc_replication.go

```diff
@@ -520,6 +522,23 @@ func (tm *TabletManager) demotePrimary(ctx context.Context, revertPartialFailure
 	}
 	defer tm.unlock()
 
+	finishCtx, cancel := context.WithCancel(context.Background())
+	defer cancel()
+	go func() {
+		select {
+		case <-finishCtx.Done():
+		// Finished running DemotePrimary. Nothing to do.
+		case <-time.After(10 * topo.RemoteOperationTimeout):
```

Comment:
why are we using `RemoteOperationTimeout` here?

Reply:
I was expecting something more predictable like hardcoded 10 minutes. having this be a variable users can tune is nice, but if we generally expect to throw away demotion-stalled tablets after 10 minutes, and a user has set `RemoteOperationTimeout` to something much shorter or longer for unrelated reasons (e.g. to influence tx throttler), we could be in for a surprise here.

Reply:
The idea is that we wait 10 times the amount of time that DemotePrimary has to run. `RemoteOperationTimeout` is a global configuration that the user tweaks to set the amount of time any RPC has. So I think having 10 times that amount should be a good value to wait for